src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
for logs column we just render 
```
if (rowOptions.isParent) {
  // Because elements are just stacked we need a spacer
  return <span>&nbsp;</span>;
}
```
a bit uncomfortable to style all the divs with `> div`